.github/workflows/cypressTestsHelpers/getEnvironmentVariables.js

```diff
@@ -0,0 +1,90 @@
+const { Command } = require("commander");
+const core = require("@actions/core");
+const fetch = require("node-fetch");
+const { Octokit } = require("@octokit/core");
+
+const program = new Command();
+
+program
+  .name("Approve PR")
+  .description("Approve and merge PR if patch release")
+  .option("--version <version>", "version of a project")
+  .option("--token <token>", "token for login to cloud")
+  .option("--repo_token <repo_token>", "github token")
+  .option("--project <project>", "release project")
+  .action(async options => {
+    const isOldVersion = await checkIfOldVersion(options.version, options.repo_token);
+    core.setOutput("IS_OLD_VERSION", isOldVersion);
+
+    core.setOutput("url", `https://v${getFormattedVersion(options.version)}.staging.saleor.cloud/`);
+
+    const branch = await getBranch(options.repo_token, options.version);
+    core.setOutput("branch", branch);
+  })
+  .parse();
+
+function getFormattedVersion(version) {
+  const regex = /^\d+\.\d+\./;
+  return version.match(regex)[0].replace(/\./g, "");
+}
```

Comment:
some error handling could be added there if for some reason this is run on a wrong branch
```suggestion
function getFormattedVersion(version) {
  const regex = /^\d+\.\d+\./;
  const match = version.match(regex);
  if (!match) throw new Error("Invalid version format");
  return match[0].replace(/\./g, "");
}
```